README.md

```diff
@@ -2,7 +2,61 @@
 [![codecov](https://codecov.io/gh/allenai/allennlp/branch/master/graph/badge.svg)](https://codecov.io/gh/allenai/allennlp)
 # AllenNLP
 
-A [Apache 2.0](https://github.com/allenai/allennlp/blob/master/LICENSE) natural language processing toolkit using state-of-the-art deep learning models.
+An [Apache 2.0](https://github.com/allenai/allennlp/blob/master/LICENSE) NLP research library, built on PyTorch,
+for developing state-of-the-art deep learning models.
```

Comment:
"for developing state-of-the-art NLP models using deep learning on a wide variety of linguistic tasks."